boilerplate/.maestro/FavoritePodcast.yaml

```diff
@@ -0,0 +1,34 @@
+# flow: run the login flow and then navigate to the demo podcast list screen, favorite a podcast, and then switch the list to only be favorites.
+
+appId: com.helloworld
+env:
+  TITLE: "RNR 257 - META RESPONDS! How can we improve React Native, part 2"
+  FAVORITES_TEXT: "Switch on to only show favorites"
+
+---
+- runFlow: Login.yaml
+- tapOn: "Podcast, tab, 3 of 4"
+- assertVisible: "React Native Radio episodes"
+- tapOn:
+    text: ${FAVORITES_TEXT}
+- assertVisible: "This looks a bit empty"
+- tapOn:
+    text: ${FAVORITES_TEXT}
+- scrollUntilVisible:
+    element:
+      text: ${TITLE}
+    direction: DOWN
+    timeout: 50000
+    speed: 40
+    visibilityPercentage: 100
+- longPressOn: ${TITLE}
+- scrollUntilVisible:
+    element:
+      text: ${FAVORITES_TEXT}
+    direction: UP
+    timeout: 50000
+    speed: 40
+    visibilityPercentage: 100
+- tapOn:
+    text: ${FAVORITES_TEXT}
+- assertVisible: ${TITLE}
```

Comment:
That type of comment won't work in a yaml file, will # @demo remove-file get picked up or do I need to make edits elsewhere?

I think I'll have to update the `demoMarkupRegex` in `tools/demo.ts`

Reply:
No that style of comment should work. I wrote it to look for just the substring, so that we could use multiple styles of comments.

<img width="428" alt="Screenshot 2023-02-28 at 4 56 55 PM" src="https://user-images.githubusercontent.com/37849890/222017508-ca55e605-08e2-4c90-ace2-bf250298a347.png">

Reply:
Threw up a commit to fix this since I was already messing around with it locally